### PR TITLE
Add GitLab

### PR DESCRIPTION
**Pull request by JobV**

### CODE REVIEW

File: _data/projects/gitlab.yml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+name: GitLab 
+desc: Code, test, and deploy together
+site: https://about.gitlab.com
+tags:
+- oss
+- git
+- issues
```
Comment (DouweM) [inline _data/projects/gitlab.yml:7]: Let's add `Ruby`, `Ruby on Rails`, `JS` and `web` :)

File: _data/projects/gitlab.yml [hunk 2]
```diff
@@ -0,0 +1,10 @@
...
+upforgrabs:
+  name: up-for-grabs
+  link: https://gitlab.com/gitlab-org/gitlab-ce/issues?milestone_id=&scope=all&sort=created_desc&state=opened&utf8=%E2%9C%93&assignee_id=&author_id=&milestone_title=&label_name=up-for-grabs
```
Comment (DouweM) [inline _data/projects/gitlab.yml]: This should be plenty: https://gitlab.com/gitlab-org/gitlab-ce/issues?label_name=up-for-grabs

File: _data/projects/gitlab.yml (hunk 1 above)
Comment (ZJvandeWeg) [reply _data/projects/gitlab.yml:7]: Maybe Vue even?

**Comment by dahlbyk** [comment]
Thanks for the fixes, @jrusbatch!

**Comment by dahlbyk** [comment]
FYI: https://github.com/up-for-grabs/up-for-grabs.net/issues/462

**Comment by tannakartikey** [comment]
up-for-grabs shows as deprecated on Gitlab official website.

**Comment by ZJvandeWeg** [comment]
We're unifying it with the Accepting Merge Requests label. Please use that! :)

**Pull request merged**